src/libUtils/Logger.cpp

```diff
@@ -81,17 +83,21 @@ void Logger::newLog() {
            m_seqNum);
   m_fileName = m_fileNamePrefix + buf;
 
+  if (!boost::filesystem::exists(LOG_PATH)) {
```

Comment:
I think we can move this to the constructor, so that it doesn't have to be checked every time the log rolls over to a new file.

Reply:
ok